internal/schema/1.5/check.go

```diff
@@ -52,7 +52,7 @@ func checkBlock() *schema.BlockSchema {
 func scopedDataBlock() *schema.BlockSchema {
 	bs := &schema.BlockSchema{
 		// TODO: Address: &schema.BlockAddrSchema{},
-		// See https://github.com/hashicorp/terraform-schema/issues/234
+		// See https://github.com/opentofu/opentofu-schema/issues/234
```

Comment:
This url doesn't exist, separate repo, separate issues.

Reply:
Thanks! fixed that https://github.com/opentofu/opentofu-schema/pull/3/commits/bf10a5718c422ad0ec99a128720ff02437d42a98